Modules/Scripted/ExtensionWizard/ExtensionWizard.py

```diff
@@ -384,25 +387,25 @@ def loadModules(path=None, depth=1, modules=None, parent=None):
 
                 if len(failed):
                     if len(failed) > 1:
-                        text = "The following modules could not be registered:"
+                        text = _("The following modules could not be registered:")
                     else:
-                        text = "The '%s' module could not be registered:" % failed[0].key
+                        text = _("The '%s' module could not be registered:") % failed[0].key
```

Comment:
Use named placeholders

Reply:
Can you explain more this?

Reply:
We use placeholders everywhere where we need to insert some value into the string. For example, the code here should look something like this:

    text = _("The {moduleName} module could not be registered").format(moduleName=failed[0].key)